### ISSUE TITLE
Healthcheck not working because wget is not installed in the docker image

### ISSUE THREAD

**Issue by zertrin** [bpatrik/pigallery2#193]
**Describe the bug**
Using the latest nightly docker image, I noticed that the health check was not working properly, simply because there is no `wget` in the image.

Proposed solution: add `wget` inside the docker image.

**Screenshots**

![image](https://user-images.githubusercontent.com/1447163/96896308-ef3d1980-14bf-11eb-8f83-64590f7b0c3b.png)

**Environment (please complete the following information):**
 - OS: Host:  Linux Debian buster, running pigallery2 in docker without using docker-compose (because I already have a reverse proxy to terminate SSL on the host).

Here is my docker run command for reference:

```bash
#!/bin/bash

TAG="nightly-stretch"
DATA_ROOT_PATH="/data/pigallery2"
IMAGES_PATH="/data/redacted"

docker run -d --name pigallery2 --restart always \
    -p 8314:80 \
   -e NODE_ENV=production \
   -v "${DATA_ROOT_PATH}/config":/app/data/config \
   -v "${DATA_ROOT_PATH}/db":/app/data/db \
   -v "${IMAGES_PATH}":/app/data/images:ro \
   -v "${DATA_ROOT_PATH}/temp":/app/data/tmp \
   bpatrik/pigallery2:${TAG}
```

**Used app version**:
- docker-nightly

**Comment by bpatrik** [comment]
Thank for the report.
Just to clarify, this is what you mean, right? `RUN apt-get update && apt-get install -y ca-certificates wget` ?
It seems other had similar problems: https://github.com/nodejs/docker-node/issues/1185

**Comment by zertrin** [comment]
Yeah good find, it seems node introduced breaking changes in their docker images indeed...

I'll open a pull request later today to update the debian Dockerfile here, I think there is some small possible improvements around the `apt install` step (e.g. to reduce image size by cleaning apt cache and temporary data after installing).

**Comment by zertrin** [comment]
Side note/question: looking at the available dockerfiles, it seems there are two of them, `Dockerfile.build` seems to be the most "up-to-date" and the one used for GitHub automatic builds, and `selfcontained/Dockerfile` being apparently desynchronized from the previous one and meant for local building.

Shall I try to resynchronize `selfcontained/Dockerfile` to be closer to what `Dockerfile.build` currently does?

![image](https://user-images.githubusercontent.com/1447163/97101617-ed4ea280-16d9-11eb-816d-8294d7ae97a3.png)

**Comment by bpatrik** [comment]
Yes! If you would have time for it I would appreciate a PR. 

The setups looks such a way that the two `Dockerfile.buid` (debianand and alpine) are used by the github workflow to build the image for multiple platforms.
I abandoned the last/thirds "self contained" one a bit lately. That can be used if some one [wants to pull and build the repo on its own](https://github.com/bpatrik/pigallery2/tree/master/docker#build-the-docker-image-on-your-own). Not sure If we should still support this.

**Comment by zertrin** [comment]
> I abandoned the last/thirds "self contained" one a bit lately. That can be used if some one [wants to pull and build the repo on its own](https://github.com/bpatrik/pigallery2/tree/master/docker#build-the-docker-image-on-your-own). Not sure If we should still support this.

Well I would argue that not maintaining them makes it difficult for contributors like me to test changes in Docker after cloning the repository... Unless you intend to only develop in a "blind" way whereby you only know whether some change works or not based on the result of GitHub builds, but that's a tedious approach where many commits and push are spent blindly trying variations of changes until the GitHub pipeline finally tells you "it works". It would have been more efficient to test locally before committing and pushing several erroneous changes...

When I proposed

> I'll open a pull request later today to update the debian Dockerfile here

earlier, I did not expect to have a hard time to set up, build and test docker images on my computer. Now as I write this, I'm still struggling to get the build to work...

For your consideration in your decision whether to maintain the non-GitHub-workflow Dockerfiles. 😉
